packages/framer-motion/src/render/VisualElement.ts

```diff
@@ -732,7 +732,7 @@ export abstract class VisualElement<
             this.addValue(key, value)
         }
 
-        return value as MotionValue
+        return value!
```

Comment:
Can we avoid the cast? It can be undefined if `defaultValue` is also undefined, right?